contracts/docs/SOROBAN_CONTRACT_SPEC.md

```
@@ -0,0 +1,246 @@
+
+#### `is_winner_selected(pool_id: u64) -> bool`
+- Checks if winner has been selected.
+- No auth required.
+
+#### `get_winner(pool_id: u64) -> Option<Address>`
+- Returns the winner address if selected.
+- No auth required.
+
+## Events
+
+### `PoolCreated`
```

markdownlint-cli2 (0.22.1)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)